_test/bootstrap.sh

```diff
@@ -8,20 +8,20 @@ fi
 if [ ! -z "$CABALVER" ]; then
     export PATH=/opt/cabal/${CABALVER}/bin:$PATH
 fi
+
+# What directory is this script in?
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+deps=$(
+  cut -d: -f2 "$DIR/dependencies.txt" |
+  xargs echo |
+  tr " " "\n" |
+  sort |
+  uniq |
+  # Exclude packages included in GHC
+  grep -v 'base\|array\|containers\|time'
```

Comment:
Instead of trying to do it by yourself, you can run ``dependencies `ls exercises` ``.

After that, it would be nice to exclude the packages already installed. Maybe `ghc-pkg list` with some options can solve this.